axelor-project/src/main/resources/views/Sprint.xml

```diff
@@ -41,4 +41,27 @@
 
   </form>
 
+  <form model="com.axelor.utils.db.Wizard" title="Sprint generator"
+    name="sprint-generator-wizard-form" width="large">
+
+    <panel name="mainPanel">
+      <field name="projects" title="Projects" target="com.axelor.apps.project.db.Project"
+        type="many-to-many" widget="TagSelect" required="true" x-dirty="false" colSpan="12"/>
+      <field name="sprintPeriods" title="Sprint periods"
+        target="com.axelor.apps.project.db.SprintPeriod" type="many-to-many"
+        onSelect="action-sprint-method-set-period-domain" widget="TagSelect" required="true"
+        x-dirty="false" colSpan="12"/>
+      <button onClick="action-sprint-method-generate" name="generate" title="Generate"/>
+    </panel>
```

Comment:
We need to redo this thing, in this view set a M2M projects and fromDate, toDate, company
Need to add an action-validate as action-employee-validate-dates at the dates onChange and before the button onClick please

Remove the sprintPeriods and in the process, search for each week in fromDate toDate (will be limited to monday for fromDate and friday for toDate in another ticket) the associated period (with a check on the company too) and if the period is null, generate it and fill the sprint's periodList